lib/screens/home_screen.dart

```diff
@@ -120,11 +111,13 @@ class _HomeScreenState extends State<HomeScreen>
                 child: const Text('Cancel'),
               ),
               FlatButton(
-                onPressed: () {
+                onPressed: () async {
                   if (contentInputController.text.isNotEmpty) {
-                    todoBloc.createTodo(contentInputController.text);
+                    await todoBloc.createTodo(contentInputController.text);
                     contentInputController.clear();
```

Comment:
I think the TextEditingController should actually be inside the bloc. It seems cleaner. The the action "createTodo()" could then clear the controller, if it completes successfully. It would make the UI code cleaner and it seems like logic that could sensibly live inside the controller. However, I don't know what the best practice is.

Reply:
Will move it to the bloc. It makes sense for the logic to live there.